### PR TITLE
TSK-1605: Remove the hardcoded schema name from the indexes

### PR DESCRIPTION
**Pull request by tge20**
---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [ ] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: common/taskana-common/src/main/resources/sql/db2/taskana-schema-db2.sql [hunk 1]
```diff
@@ -294,26 +294,27 @@ CREATE SEQUENCE SCHEDULED_JOB_SEQ
...
+   ON "CLASSIFICATION" ("ID" ASC) INCLUDE
    ("CUSTOM_8", "CUSTOM_7", "CUSTOM_6", "CUSTOM_5", "CUSTOM_4",
    "CUSTOM_3", "CUSTOM_2", "CUSTOM_1", "APPLICATION_ENTRY_POINT",
    "SERVICE_LEVEL", "PRIORITY", "DESCRIPTION", "NAME",
    "CREATED", "VALID_IN_DOMAIN", "DOMAIN", "TYPE", "CATEGORY",
    "PARENT_KEY", "PARENT_ID", "KEY") ALLOW REVERSE SCANS COLLECT SAMPLED DETAILED STATISTICS;
    COMMIT WORK ;
 -- index[2],    3,646MB
-   CREATE INDEX "DB2ADMIN"."IDX1805212018000" ON "TASKANA "."TASK"
+   CREATE INDEX "DB2ADMIN"."IDX1805212018000" ON "TASK"
```
Comment (holgerhagen) [inline common/taskana-common/src/main/resources/sql/db2/taskana-schema-db2.sql:312]: "DB2ADMIN" is a schema name as well and should be removed as well.

**Review by holgerhagen — CHANGES_REQUESTED** [review]
We should name the indexes in a meaningful way.

File: common/taskana-common/src/main/resources/sql/db2/taskana-schema-db2.sql (hunk 1 above)
Comment (tge20) [reply common/taskana-common/src/main/resources/sql/db2/taskana-schema-db2.sql:312]: True, but wouldn't this mean that we create the indixes in our "TASKANA" schema as we set this as our current schema in the first line? Do we want to implement this behavior?

Comment (tge20) [reply common/taskana-common/src/main/resources/sql/db2/taskana-schema-db2.sql]: We agreed to put them in the "TASKANA" schema.

**Comment by mustaphazorgati** [comment]
@holgerhagen what about this PR? There have been no updates for almost a year now? Can we close this?

**Comment by holgerhagen** [comment]
solved with TSK-1790

**Pull request closed without merging**